extensions/amp-gist/0.1/validator-amp-gist.protoascii

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-gist
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-gist extension .js script"
+  extension_spec: {
+    name: "amp-gist"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    deprecated_recommends_usage_of_tag: "amp-gist"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-gist>
+  html_format: AMP
+  tag_name: "AMP-GIST"
+  requires: "amp-gist extension .js script"
+  attrs: {
+    name: "data-gistid"
+    mandatory: true
+    blacklisted_value_regex: "__amp_source_origin"
```

Comment:
This `blacklisted_value_regex` is unnecessary since this isn't a full url value. IOW, you append the value of `data-gistid` to a hardcoded url in amp-gist.js so we don't need to blacklist `__amp_source_origin`.